src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -10,4 +10,5 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <PackageReleaseNotes>See https://aka.ms/asdotnetsdkchangelog for release notes.</PackageReleaseNotes>
```

Comment:
This property has to be part of a propertygroup

Reply:
Updated.